### DISCUSSION TITLE
Translating $eventName (localization)

### DISCUSSION THREAD

**Discussion by AskinSavascisi** [spatie/laravel-activitylog#1021] [Q&A] [answered]
Hi,

is it possible to translate the $eventName variable to  active localization version?

public function getActivitylogOptions()
    {
        return LogOptions::defaults()
        ->setDescriptionForEvent(fn(string $eventName) => __('lang.has_been')."{$eventName}");
    }

**Comment by seche** [comment, marked as answer]
> Hi,
> 
> is it possible to translate the $eventName variable to active localization version?
> 
> public function getActivitylogOptions() { return LogOptions::defaults() ->setDescriptionForEvent(fn(string $eventName) => __('lang.has_been')."{$eventName}"); }

Just use __($eventName) and include the translation key in your translation files.

```
public function getActivitylogOptions()
{
return LogOptions::defaults()
->setDescriptionForEvent(fn(string $eventName) => __('lang.has_been') . __($eventName));
}
```

By default, the package provides the following keys: `created`, `updated`, `deleted`.

**Reply by AskinSavascisi** [reply]
İndeed, i did solve it like that
